SpatialGDK/Documentation/content/get-started/starter-template/get-started-template-local.md

```diff
@@ -41,64 +50,113 @@ You can find out more about snapshots in the [GDK snapshot documentation]({{urlR
+    This stops the running client and server-worker instances but keeps your deployment running.
 
-As you iterate on your game (i.e modifying classes or blueprints), **making changes to replicated components will require incremental schema regeneration**. To do this, select the **Schema** button in the GDK toolbar.
+### Step 4: Iterate on game development
 
-![Stop]({{assetRoot}}assets/screen-grabs/toolbar/schema-button.png)<br/>
+As you iterate on development of your game (by modifying classes or Blueprints, for example), you will make changes changes to replicated components - this requires incremental schema regeneration. 
 
-Once you've regenerated schema, the GDK will restart the running deployment with the new schema.
+To do this:
 
-If you haven't modified anything related to replication, you don't need to regenerate schema and the previously running deployment will continue to be used.
+1. Select the **Schema** button in the GDK toolbar. (Note you do not need to do a full scan schema generation for incremental changes.)
 
-To test your changes, hit play again which will start your client and server workers.
+    ![Stop]({{assetRoot}}assets/screen-grabs/toolbar/schema-button.png)<br/>
+    _Image: GDK toolbar's **Schema** button_</br></br>
 
-<%(#Expandable title="Workflow reference diagram")%>
+    Once you've regenerated schema, the GDK restarts the running deployment with the new schema.
 
- <%(Lightbox image="https://docs.google.com/drawings/d/e/2PACX-1vQCTOucXKMkDJ3-Vpg17_tpUS7IxOXD6Mps-FzWe2tQl3vw5alQPngCnw339cFy3u2NvrcBxhYASKsS/pub?w=710&h=1033")%>
+    If you haven't modified anything related to replication, you don't need to regenerate schema and SpatialOS continues to use the running deployment. </br></br>
 
-For more details, see the [Local deployment workflow page]({{urlRoot}}/content/local-deployment-workflow).
+2. To test your changes, select **Play** on the Unreal toolbar; this starts your game's clients and server-worker instances.
+
+<%(#Expandable title="Local deployment workflow summary")%>
+There is a summary on the [Local deployment workflow]({{urlRoot}}/content/local-deployment-workflow) page. It is the same as the one here.
+ <%(Lightbox image="https://docs.google.com/drawings/d/e/2PACX-1vQCTOucXKMkDJ3-Vpg17_tpUS7IxOXD6Mps-FzWe2tQl3vw5alQPngCnw339cFy3u2NvrcBxhYASKsS/pub?w=710&h=1033")%>
 
 <%(/Expandable)%>
 
 ### Step 5: Stop your deployment
 
-To fully stop your SpatialOS deployment, select **Stop** in the GDK toolbar.<br/></br>
+To fully stop your local deployment, select **Stop** in the GDK toolbar.<br/></br>
 ![Stop]({{assetRoot}}assets/screen-grabs/toolbar/stop-button.png)<br/>
+ _Image: GDK toolbar's **Stop** button_</br></br>
 
-### **> Next:** [3: Launch a cloud deployment]({{urlRoot}}/content/get-started/starter-template/get-started-template-cloud)
+#### **> Next:** [3: Launch a cloud deployment]({{urlRoot}}/content/get-started/starter-template/get-started-template-cloud)
```

Comment:
I really think it's better UX to have a larger Next link. It's too easy to miss otherwise.

Reply:
I agree. It's a mare.

They were appearing in some of the in-page ToC due to the number of levels of titles we had on a page.
I tried lots of different ways to make it standard without them appearing in the in-page ToCs and this was the most efficient way.